pkg/api/grpcServer_test.go

```diff
@@ -459,7 +466,7 @@ func TestRudeClose(t *testing.T) {
 }
 
 func TestBrokenStream(t *testing.T) {
-	ts, err := prepTestState(30000)
+	ts, err := prepTestState(30009)
```

Comment:
I feel like we are doing the wrong thing here. I don't think we should be specifying ports at all in testing, if we can avoid it. IIRC, we should be able to listen on port 0 (or unspecified) and then get the actual port from the listener addr assigned, if needed.